source/guides/clear/swupd-3rd-party.rst

```diff
@@ -0,0 +1,281 @@
+.. _swupd-3rd-party:
+
+Create, Install, and Manage 3rd-party Bundles
+#############################################
+
+Upstream Clear Linux offers a plethora of bundles to choose from.  
+As of version 32570, there are over 1000 bundles and counting.  
```

Comment:
I would drop this line and combine the whole intro into one or two paragraphs. Specific versions reference are out of date very easily. Maybe link to the software store here

Reply:
pointing to https://clearlinux.org/software